tutorials/cloudsdk-cap-pipeline-model/cloudsdk-cap-pipeline-model.md

```diff
@@ -1,232 +1,295 @@
+Create and run a build job for your project in Jenkins.
 
-To deploy applications, you need to create a free trial account. You can create your account by visiting [SAP Cloud Platform](https://cloudplatform.sap.com/try.html)
+1. From the navigation pane in Jenkins, choose **New Item**.
 
-After creating your account and activating it through email, you can log in to your personal **SAP Cloud Platform Cockpit**. For your first visit, it should look like this:
+2. Enter a name for your build job, select **Multibranch Pipeline**, and choose **OK**.
 
-![SAP Cloud Platform Cockpit first view](cloudplatform-cockpit-first-view.png)
+3. Under **Branch Sources**, choose **Add Source** &rarr; **Git**.
 
-Now click the **Home** button in the upper navigation bar and then click **Start Cloud Foundry Trial**.
+4. In your repository in GitHub, choose **Code** and copy your repository URL for HTTPS.
 
-![SAP Cloud Platform Cockpit start trial view](cloudplatform-cockpit-start-cf-trial1.png)
+5. In Jenkins, enter your copied repository URL as **Project Repository** and choose **Save**.
 
-After selecting your region, your account will be automatically set up for development with **Cloud Foundry**.
+    As a result, Jenkins scans your GitHub repository and triggers a build.
 
-![Cloud foundry view](cftrial.png)
+6. To view the status of your build, choose **Open Blue Ocean** from the navigation pane.
 
-In order to deploy applications on **SAP Cloud Foundry** you need to provide the API endpoint and other properties. You can get them by going inside your trial.
+7. Click on your job to see a detailed log of your job as well as a graphical representation of your pipeline.
 
-![Cloud foundry properties](cloudfoundry.png)
+    When the build job has finished successfully, your pipeline should look as follows:
 
-Configure `config.yml` for Cloud Foundry settings.
+    !![Stages of the SAP Cloud SDK pipeline without production deployment](pipeline-without-prod.png)
 
-```
-#config.yml
-productionDeployment:
-  cfTargets:
-   - space: 'my-space'
-     org: 'my-org'
-     credentialsId: 'credentials'
-     apiEndpoint: '<Cloud Foundry API endpoint>'
-```
 
-Now you require credentials. It can be created in the Jenkins by navigating to **Credentials > System > Global Credentials**.
+[VALIDATE_1]
+[ACCORDION-END]
 
-!![Jenkins System](jenkins-system.png)
+[ACCORDION-BEGIN [Step 5: ](Add production deployment to your CI/CD pipeline)]
 
-On this screen, click **Add Credentials** and enter username, password, as well as an ID for your credentials record.
+Configure the *Production Deployment* stage of your CI/CD pipeline.
 
-You can now leverage this credentials record by adding the credentials configuration property to the `productionDeployment` stage of your `config.yml`.
+1. In your SAP Cloud Platform trial account, click on your subaccount tile.
```

Comment:
Don't say "click on". Say "click the <insert thing to be clicked>".

Reply:
Done.